libr/core/cbin.c

```diff
@@ -796,6 +796,9 @@ static int bin_info(RCore *r, int mode, ut64 laddr) {
 			if (info->cpu && *info->cpu) {
 				r_config_set (r->config, "asm.cpu", info->cpu);
 			}
+			if (info->features && *info->features) {
```

Comment:
FYI we have `R_STR_ISNOTEMPTY` and `R_STR_ISEMPTY` to do these kind of checks. Just wanted to mention it :) I know a lot of the existing code still uses `p && *p`.

Reply:
okay, I just want to keep the same style as the code here